BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,313 @@
+     * Then, verifies if the image of the product is visible.
+     */
+    public void verifyProdImage() {
+    	this.visitPDP();
+        extentInfoLog("Image is displayed : ", Action.isDisplayed(driver, prodImg));
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the title of the product.
+     */
+    public void verifyProdTitle() {
+    	this.visitPDP();
+        extentInfoLog("Title is displayed : ", Action.isDisplayed(driver, prodTitle));
+        extentInfoLog("Product title verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies if the rating of the product is visible.
+     */
+    public void verifyProdRating() {
+    	this.visitPDP();
+        extentInfoLog("Rating is displayed : ", Action.isDisplayed(driver, prodRating));
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies if the overview of the product is visible.
+     */
+    public void verifyProdOverview() {
+    	this.visitPDP();
+        extentInfoLog("Overview is displayed : ", Action.isDisplayed(driver, prodOverview));
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the buttons.
+     */
+    public void verifyButtons(String expectedText) {
+        if(expectedText.contains("Write a review")) {
+            this.verifyWriteAReviewBtn(expectedText);
+        } else {
+        	this.verifyBuyNowBtn(expectedText);
+        }
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the Write a Review button.
+     */
+    public void verifyWriteAReviewBtn(String expectedText) {
+    	this.visitPDP();
+        extentInfoLog("Button is displayed : ", Action.isDisplayed(driver, writeAReviewBtn));
+		String actualText= writeAReviewBtn.getText();
+        Assert.assertEquals(actualText, expectedText);
+		Action.performActionwithExtentInfoLog(writeAReviewBtn, "click", "Clicking on : " + actualText);
+        extentInfoLog("Modal is displayed : ", Action.isDisplayed(driver, writeAReviewModal));
+        extentInfoLog("Write a Review button verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the Buy Now button.
+     */
+    public void verifyBuyNowBtn(String expectedText) {
+    	this.visitPDP();
+        extentInfoLog("Button is displayed : ", Action.isDisplayed(driver, buyNowBtn));
+		String actualText = buyNowBtn.getText();
+        Assert.assertEquals(actualText, expectedText);
+		Action.performActionwithExtentInfoLog(buyNowBtn, "click", "Clicking on : " + actualText);
+        extentInfoLog("Modal is displayed : ", Action.isDisplayed(driver, buyNowModal));
+        extentInfoLog("Buy Now button verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the jump to heading.
+     */
+    public void verifyJumpToOverview(String jumpToHeading, String sectionHeading) {
```

Comment:
@zahinSJI all the below functions which starts with jumpto is not needed. You can just write 1 function for all. Modify accordingly.

Reply:
Added as suggested